tests/models/compute/security_group_tests.rb

```diff
@@ -35,6 +35,18 @@
       @group.ip_permissions.empty?
     end
 
+    test("authorize access at a port range (egress rule)") do
+      @group.authorize_port_range(5000..6000, :direction => 'egress')
+      @group.reload
+      @group.ip_permissions_egress.size == 1
+    end
+
+    test("revoke access at a port range (egress rule)") do
+      @group.revoke_port_range(5000..6000, :direction => 'egress')
+      @group.reload
+      @group.ip_permissions_egress.empty?
```

Comment:
similar to https://github.com/fog/fog-aws/pull/290/files#r75165909, i would recommend asserting the absence of the specified rule.